.kitchen.yml

```diff
@@ -8,16 +8,25 @@ driver:
 
 provisioner:
   name: chef_zero
+  require_chef_omnibus: true
+  always_update_cookbooks: true
+
+verifier:
+  name: inspec
+  sudo: true
 
 platforms:
   - name: centos-6.6
-    driver_config:
-      box: sendgrid_centos-6_chef-11
-      box_url: http://repo.sjc1.sendgrid.net/images/vagrant/metadata/sendgrid_centos-6_chef-11.json
+    attributes:
+      orchestrator:
+        package:
+          version: 2.1.5-1
+          url: 'https://github.com/kplimack/orchestrator-rpm/raw/master/orchestrator-2.1.5-1.x86_64.rpm'
```

Comment:
did we get shlomi to re-upload this, @silviabotros ?

Reply:
Also at https://github.com/silviabotros/chef-orchestrator/pull/9/files#diff-c36c03b7f2ee2bb67c50a7c931a63a0eR22

Reply:
nope...Shlomi is on pro till next week....we are on our own with the package till he resolves the build issues in 3.x